### ISSUE TITLE
Game Location and Status in Games on Homepage

### ISSUE THREAD

**Issue by Whompratt** [tobymao/18xx#9913]
**What's your idea?**

Currently, if someone doesn't know the location setting or status of a game when browsing new and ongoing games, they have to either refer to github wiki, or use the 'create a new game' ui to see the list of games.
Making this information visible when viewing new and active games would make it easier to decide whether to join a game.

I think having this as a tooltip on the new game widget, or even just in the widget itself would be useful.
Given status is either Production, Beta, Alpha, or Prototype, a simple (P), (B/β), (A/α), (Pt) would make choosing to join a game more informative.

-----

Basically, make this info:
![image](https://github.com/tobymao/18xx/assets/2126241/c2dbac44-92d6-484f-83c3-d79b03823320)
Visible here:
![image](https://github.com/tobymao/18xx/assets/2126241/1ddef076-a047-4369-aa1d-cfa29284ed70)

**Are there related ideas?**

A quick scan over the open feature requests doesn't show any similar requests. Apologies if this is a duplicate.

**Comment by philcampeau** [comment]
Discussed with other devs. Most agree the game location is unnecessary, but we'll add the development status of the game.

**Comment by Whompratt** [comment]
Could I also suggest potentially embedding a link to the relevant GitHub page, for a way to quickly reference the game in question?

This kind of info is mostly beneficial to less knowledgeable players like myself, but the downside is a more cluttered interface.

**Comment by crericha** [comment]
The GitHub page is in the info tab. Does that satisfy the request?

**Comment by philcampeau** [comment]
The problem there is you can't access the info tab until the game is started. 

We could potentially make the game title link to the wiki page of each game.

**Comment by scottredracecar** [comment]
I think if the game is in alpha/beta status, the alpha/beta word could link
to the known issues.

https://github.com/tobymao/18xx/issues?q=is%3Aissue+is%3Aopen+label%3A1850

**Comment by Whompratt** [comment]
The goal of my suggestion is to make it easier to access relevant information when deciding whether or not to join a game, which the alpha/beta flag helps with, but things like checking the map or the rules currently requires either finding the game in question in the game creation UI, which is a separate page, or finding the game in the Github wiki, which requires a second tab.

Having the relevant wiki page linked under the game title gives the option to open directly. Not necessary, but definitely valuable.
